frontend/src/views/ShareSecretPublicPage/components/ShareSecretForm.tsx

```diff
@@ -85,18 +85,22 @@ export const ShareSecretForm = ({ isPublic, value }: Props) => {
         accessType
       });
 
-      setSecretLink(`${window.location.origin}/shared/secret/${id}`);
+      const link = `${window.location.origin}/shared/secret/${id}`;
```

Comment:
I wouldn't make this a variable, I would just do `setSecretLink(${window.location.origin}/shared/secret/${id})`